src/app/repository/repository.service.ts

```diff
@@ -21,11 +21,27 @@ export class DataRepository {
   }
 
   getMyGames(): Observable<Game[]> {
-    return of(GAMES_HISTORY).pipe(delay(1000));
+    return of(GAMES_HISTORY).pipe(
+      delay(1000),
+      map((games) =>
+        games.map((game) => ({ ...game, startAt: new Date(game.startAt) })),
+      ),
+    );
   }
 
   setBet(amount: number): Observable<null> {
     console.info('setBet with value', amount);
     return of(null).pipe(delay(500));
   }
+  getCurrentGame(): Observable<Game> {
+    // returns the game that started 20 seconds ago
+    const now = Date.now();
+    const twentySecondsAgo = now - 20000;
```

Comment:
I would suggest to use random here so user can test both use-cases

Reply:
done